### ISSUE TITLE
Can't connect to a server with latest version + Serene Seasons

### ISSUE THREAD

**Issue by raffle2** [Snownee/SnowRealMagic#399]
### Mod loader

Forge

### Minecraft version

1.20.1

### Mod version

10.5.2

### Modloader version

Forge 47.3.0

### [IMPORTANT] If bug:

- [X] I have confirmed this bug is reproducible on a minimal setup, not a modpack with many other mods.

### [IMPORTANT] If bug: The **latest.log** file, not the crash report

https://mclo.gs/OHgNN2S

### Issue description

Hello,

I’m encountering an issue where I'm unable to connect to a server when using your latest version in combination with the latest versions of Serene Seasons. I don't receive a crash report or error message... just repeated "Timed out" messages when attempting to connect.

Here’s a link to my mod list (it's very lightweight):
![O5VHCAU](https://github.com/user-attachments/assets/231b2336-40d9-41b3-abfa-05333a4669d6)

And here’s the server log: https://mclo.gs/OHgNN2S (I didn’t notice anything significant myself).

The server doesn't let me connect under these conditions when using the following mods together:

- SereneSeasons-forge-1.20.1-9.1.0.0
- sereneseasonsfix-1.20.2-1.1.0.1
- SnowRealMagic-1.20.1-Forge-10.5.2

However, everything works fine if I revert to the following versions:

- SereneSeasons-forge-1.20.1-9.0.0.46
- sereneseasonsfix-1.20.1-1.0.8
- SnowRealMagic-1.20.1-Forge-10.5.1

Alternatively, the server also works if I use the latest versions of just the serene seasons, or your mod alone and not together.

Thank you for reading! :)

**Comment by Snownee** [comment]
try removing sereneseasonsfix. also you sent two identical links

**Comment by raffle2** [comment]
It seems that after removing the Serene Seasons fix, I was able to connect without any issues. Strangely enough, I re-added it afterward and could still connect just fine too. It looks like I only needed to remove it for the initial server load after updating both Serene Seasons and your mod. 

I’m not sure why, but I’m glad it's working now. 

Sorry for taking up some of your time, but thanks for the help!

**Comment by raffle2** [comment]
Actually, never mind. Thinking that was the solution, I went ahead and tried it on my main server. I removed the Serene Seasons fix and updated both Serene Seasons and your mod to their latest versions, but I'm still encountering the same timeout issue. 

However, this time I do see a relevant error message that could be related to the problem, possibly tied to the generateSnowAndIce event towards the bottom. https://mclo.gs/UVk8ycq 

Could this be the cause?
